x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_connect/engine_connect.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues } from 'kea';
+
+import { i18n } from '@kbn/i18n';
+
+import { EngineViewTabs, SearchApplicationConnectTabs } from '../../../routes';
+import { EnterpriseSearchEnginesPageTemplate } from '../../layout/engines_page_template';
+
+import { EngineViewLogic } from '../engine_view_logic';
+
+import { SearchApplicationAPI } from './search_application_api';
+
+const pageTitle = i18n.translate(
+  'xpack.enterpriseSearch.content.searchApplications.connect.pageTitle',
+  {
+    defaultMessage: 'Connect',
+  }
+);
+
+export const EngineConnect: React.FC = () => {
+  const { engineName, isLoadingEngine } = useValues(EngineViewLogic);
+  const [connectTabId, setTab] = React.useState<SearchApplicationConnectTabs>(
```

Comment:
I want to make this a path parameter, but when I did the Side Nav didn't render the search application or it's children.

Investigating why that is now, but didn't want to hold up this work if I can't figure it out.